src/Banking/Statement.php

```diff
@@ -35,6 +41,15 @@ public function getOldBalance(): ?Balance
         return $this->oldBalance;
     }
 
+    public function getOldBalanceOrThrowException(): Balance
+    {
+        if ($this->oldBalance === null) {
+            throw new \RuntimeException('old balance is null');
```

Comment:
We could add a dedicated exception for this, WDYT?

Reply:
i've added BalanceUnavailableException